app/models/ecf_participant_eligibility.rb

```diff
@@ -21,6 +21,7 @@ class ECFParticipantEligibility < ApplicationRecord
     different_trn: "different_trn",
     duplicate_profile: "duplicate_profile",
     none: "none",
+    no_induction: "no_induction",
```

Comment:
I might be missing something but should this be used in the `determine_status` method?  What should/will happen if that flag is set and `determine_status` is called?

Reply:
🤦 Thanks, that was the last thing left to do which I forgot about